src/pages/cycles/[id].js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import CycleSection from '@components/cycle-section/cycle-section/CycleSection';
+import { Skeleton } from 'primereact/skeleton';
+
+const CyclePage = () => {
+  const { query, isReady } = useRouter();
+
+  if (!isReady)
+    return <Skeleton shape="rectangle" width="100%" height="100%" />;
+
+  const cycleId = query.id;
+
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
What do you want to do here?

Reply:
Not show this page in production until is ready